.github/workflows/test_install.yml

```diff
@@ -0,0 +1,68 @@
+name: Test Installation
```

Comment:
```suggestion
name: E2E Release Installation Test
```

Reply:
Renamed to `E2E Test Release Installation`